tests/virt_autotest/login_console.pm

```diff
@@ -35,9 +35,12 @@ sub login_to_console() {
     assert_screen([qw(grub2 grub1)], 120);
     if (!get_var("reboot_for_upgrade_step")) {
         if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
-            send_key_until_needlematch("virttest-bootmenu-xen-kernel", 'down', 10, 1);
-            send_key 'ret';
+            if (!match_has_tag("virttest-bootmenu-xen-kernel")) {
+                send_key 'down';
```

Comment:
I agree with @dzedro , these changes do not make any sense to me

The solution to the problem with menu entry selection requires something like line 51-53 in installation/grub_test.pm where we do that to stop the timeout so we have more time for the needles to match

Arbitrarily pressing down is NOT a good solution and will break things when boot menu orders change